tests/functional/smoke_tests/features/steps/dashboard_page.py

```diff
@@ -0,0 +1,28 @@
+from behave import then, when
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+@when('They click on an collection exercise period')
+def click_on_collex(context):
+    WebDriverWait(context.browser, 10).until(
+        EC.visibility_of_element_located((By.ID, 'collex-datatable')))
```

Comment:
We should do this the same way its done for the other one (data attributes) for consistency?. If id changed for whatever reason, the test should still function.

Reply:
disagree

Reply:
agree